### ISSUE TITLE
Program blocked by Symantec AntiVirus

### ISSUE THREAD

**Issue by sunraysir** [AlessandroZ/LaZagne#110]
Program blocked by Symantec AntiVirus

**Comment by sunraysir** [comment]
Program blocked by Symantec AntiVirus Version 12.1.6

**Comment by AlessandroZ** [comment]
I won't fix this problem, check other issue (https://github.com/AlessandroZ/LaZagne/issues/112, etc.)
